ai_agents/agents/ten_packages/extension/google_tts_python/extension.py

```
@@ -157,55 +157,49 @@ def _reset_request_state(self) -> None:
+                self.ten_env.log_info(
+                    "Flushing Google TTS client - cleaning old connection"
+                )
+                self.client.clean()  # Clean up old connection first
 
-            # Check if ten_env is available before calling handle_completed_request
+                await self.client.reset()  # Initialize new connection
+            else:
+                self.ten_env.log_warning(
+                    "Client is not initialized, skipping reset"
+                )
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)